app/page.tsx

```diff
@@ -2,100 +2,56 @@ import Image from "next/image";
-          <li className="mb-2">
-            Get started by editing{" "}
-            <code className="bg-black/[.05] dark:bg-white/[.06] px-1 py-0.5 rounded font-semibold">
-              app/page.tsx
-            </code>
-            .
-          </li>
-          <li>Save and see your changes instantly.</li>
-        </ol>
+    <div className="min-h-screen p-8 pb-20 gap-16 sm:p-20 font-[family-name:var(--font-geist-sans)]">
+      <main className="max-w-4xl mx-auto">
+        <div className="text-center space-y-6 mb-12">
+          <h1 className="text-4xl sm:text-6xl font-bold tracking-tight">
+            Your Dog's Personal Chauffeur
+          </h1>
+          <p className="text-xl text-muted-foreground">
+            Safe, reliable transportation for your four-legged family members.
+            From vet visits to grooming appointments, we've got your pup
+            covered.
+          </p>
+        </div>
+
+        <div className="grid grid-cols-1 sm:grid-cols-2 gap-8 mb-16">
+          <div className="p-6 border rounded-lg">
+            <h2 className="text-2xl font-semibold mb-4">For Pet Parents</h2>
+            <ul className="space-y-3">
+              <li>✓ Real-time tracking of your dog's journey</li>
+              <li>✓ Vetted and trained pet transport specialists</li>
+              <li>✓ Flexible scheduling for routine appointments</li>
+              <li>✓ Safe, comfortable, climate-controlled vehicles</li>
+            </ul>
+          </div>
+
+          <div className="p-6 border rounded-lg">
+            <h2 className="text-2xl font-semibold mb-4">How It Works</h2>
+            <ul className="space-y-3">
+              <li>1. Book a ride through our easy-to-use app</li>
+              <li>2. Meet your certified pet transport specialist</li>
+              <li>3. Track your dog's journey in real-time</li>
+              <li>4. Receive updates throughout the trip</li>
+            </ul>
+          </div>
+        </div>
 
-        <div className="flex gap-4 items-center flex-col sm:flex-row">
+        <div className="flex justify-center gap-4">
           <a
-            className="rounded-full border border-solid border-transparent transition-colors flex items-center justify-center bg-foreground text-background gap-2 hover:bg-[#383838] dark:hover:bg-[#ccc] text-sm sm:text-base h-10 sm:h-12 px-4 sm:px-5"
-            href="https://vercel.com/new?utm_source=create-next-app&utm_medium=appdir-template-tw&utm_campaign=create-next-app"
-            target="_blank"
-            rel="noopener noreferrer"
+            href="#"
+            className="rounded-full bg-foreground text-background px-8 py-3 font-semibold hover:bg-foreground/90"
           >
-            <Image
-              className="dark:invert"
-              src="/vercel.svg"
-              alt="Vercel logomark"
-              width={20}
-              height={20}
-            />
-            Deploy now
+            Book a Ride
```

Comment:
These anonymous hrefs gotta be replaced fr. Add the actual paths or targets you're pointing to or else it's just a dead end 😩.